src/VirtoCommerce.SubscriptionModule.Web/VirtoCommerce.SubscriptionModule.Web.csproj

```diff
@@ -21,10 +21,25 @@
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="Microsoft.AspNetCore.Mvc.NewtonsoftJson" Version="3.1.2" />
+    <PackageReference Include="VirtoCommerce.NotificationsModule.TemplateLoader.FileSystem" Version="3.7.0" />
     <PackageReference Include="VirtoCommerce.OrdersModule.Core" Version="3.1.0" />
   </ItemGroup>
   <ItemGroup>
     <ProjectReference Include="..\VirtoCommerce.SubscriptionModule.Core\VirtoCommerce.SubscriptionModule.Core.csproj" />
     <ProjectReference Include="..\VirtoCommerce.SubscriptionModule.Data\VirtoCommerce.SubscriptionModule.Data.csproj" />
   </ItemGroup>
+  <ItemGroup>
```

Comment:
Please check templates will be kept after publication and included in artifacts
To include templates use the following code in Web csproj
```

<ItemGroup>
<NotificationTemplates Include="NotificationTemplates/**"/>
</ItemGroup>    
<Target Name="CopyCustomContentOnPublish" AfterTargets="Publish">
  <Copy SourceFiles="@(NotificationTemplates)" DestinationFiles="$(PublishDir)\..\%(Identity)" />
</Target>
```
Take reference to https://docs.microsoft.com/en-us/visualstudio/msbuild/msbuild-well-known-item-metadata?view=vs-2019